[2024-07-25T09:17:12.797+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [queued]>
[2024-07-25T09:17:12.938+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [queued]>
[2024-07-25T09:17:12.940+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-25T09:17:12.992+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_id_from_json> on 2023-04-04 00:00:00+00:00
[2024-07-25T09:17:13.027+0000] {standard_task_runner.py:57} INFO - Started process 544 to run task
[2024-07-25T09:17:13.042+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_ids_ingest', 'get_id_from_json', 'scheduled__2023-04-04T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/ids_ingest/ids_ingest_dag.py', '--cfg-path', '/tmp/tmps0rwivie']
[2024-07-25T09:17:13.106+0000] {standard_task_runner.py:85} INFO - Job 34: Subtask get_id_from_json
[2024-07-25T09:17:13.331+0000] {task_command.py:416} INFO - Running <TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [running]> on host c7799c7b21e5
[2024-07-25T09:17:13.779+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ajay' AIRFLOW_CTX_DAG_ID='simple_ids_ingest' AIRFLOW_CTX_TASK_ID='get_id_from_json' AIRFLOW_CTX_EXECUTION_DATE='2023-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-04T00:00:00+00:00'
[2024-07-25T09:17:13.784+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ids_ingest/ids_ingest_dag.py", line 18, in get_ids_from_json
    with open(filename_json, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/files_to_test/github_events.json'
[2024-07-25T09:17:13.826+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=simple_ids_ingest, task_id=get_id_from_json, execution_date=20230404T000000, start_date=20240725T091712, end_date=20240725T091713
[2024-07-25T09:17:13.913+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 34 for task get_id_from_json ([Errno 2] No such file or directory: '/opt/airflow/files_to_test/github_events.json'; 544)
[2024-07-25T09:17:13.976+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-25T09:17:14.004+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-25T14:21:51.805+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [queued]>
[2024-07-25T14:21:52.063+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [queued]>
[2024-07-25T14:21:52.064+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-25T14:21:52.297+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_id_from_json> on 2023-04-04 00:00:00+00:00
[2024-07-25T14:21:52.432+0000] {standard_task_runner.py:57} INFO - Started process 3359 to run task
[2024-07-25T14:21:52.523+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_ids_ingest', 'get_id_from_json', 'scheduled__2023-04-04T00:00:00+00:00', '--job-id', '350', '--raw', '--subdir', 'DAGS_FOLDER/ids_ingest/ids_ingest_dag.py', '--cfg-path', '/tmp/tmpp4z8fmji']
[2024-07-25T14:21:52.546+0000] {standard_task_runner.py:85} INFO - Job 350: Subtask get_id_from_json
[2024-07-25T14:21:52.943+0000] {task_command.py:416} INFO - Running <TaskInstance: simple_ids_ingest.get_id_from_json scheduled__2023-04-04T00:00:00+00:00 [running]> on host b7334bec8397
[2024-07-25T14:21:53.654+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ajay' AIRFLOW_CTX_DAG_ID='simple_ids_ingest' AIRFLOW_CTX_TASK_ID='get_id_from_json' AIRFLOW_CTX_EXECUTION_DATE='2023-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-04T00:00:00+00:00'
[2024-07-25T14:21:53.730+0000] {logging_mixin.py:154} INFO - ['25208138097', '25208138110', '25208138076', '25208138082', '25208138077', '25208138087', '25208138078', '25208138069', '25208138072', '25208138080', '25208138070', '25208138067', '25208138068', '25208138066', '25208138047', '25208138051', '25208138053', '25208138060', '25208138071', '25208138061', '25208138030', '25208138044', '25208138043', '25208138038', '25208138032', '25208138018', '25208138016', '25208138012', '25208138008', '25208137998']
[2024-07-25T14:21:53.734+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-07-25T14:21:53.832+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_ids_ingest, task_id=get_id_from_json, execution_date=20230404T000000, start_date=20240725T142151, end_date=20240725T142153
[2024-07-25T14:21:54.075+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-25T14:21:54.321+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
